refactoringminer/src/main/java/gr/uom/java/xmi/UMLModelASTReader.java

```diff
@@ -346,12 +343,12 @@ private String getAnonymousCodePath(PsiAnonymousClass anonymous) {
             if (parent instanceof PsiMethod) {
                 String methodName = ((PsiMethod) parent).getName();
                 elements.addFirst(methodName);
-            } else if (parent instanceof PsiField) {
-                String fieldName = ((PsiField) parent).getName();
+            } else if (parent instanceof PsiVariable) {
```

Comment:
please, clarify the cases when fields have type `PsiVariable` instead of `PsiField`

Reply:
When it is local variable. For example in this case:
```
var tmp = process(new Supertype() {/* some code here */});
```
Suffix of the anonymous code path expected to be `.tmp.process`